### ISSUE TITLE
Update Click to Call Common Component to always include TTY number

### ISSUE THREAD

**Issue by msmanak** [department-of-veterans-affairs/va.gov-team#17022]
# User Story
- [x] As a user of the VA mobile app, I want to be able to call my provider via TTY number within the app
- [x] As a user of the VA mobile app who uses TTY, I want to pinpoint which number is the correct number to call

# Tasks
- [x]  Find the correct TTY Icon
- [x]  Update design library component to include TTY component in Click to Call 
- [x]  Identify all the places in the app this change should be made

## new screen/component
https://adhoc.invisionapp.com/share/AX108RJZPB6E#/screens/445194522

![phone-call Copy.svg](https://images.zenhubusercontent.com/5faabf11b099434179a66d66/7e0d17a7-5243-44cc-893f-7efeef9a20c0)
![Letters (Empty).png](https://images.zenhubusercontent.com/5faabf11b099434179a66d66/a01aeb3c-fe4c-40fd-9231-9820b02c1de2)!

* see comments for placement of where 711 should be as well updated content.

## existing screen
![VA Appointment.png](https://images.zenhubusercontent.com/5faabf11b099434179a66d66/ff9fc152-6f5f-4fde-bfeb-abdb8811bb67)

![Contact VA.png](https://images.zenhubusercontent.com/5faabf11b099434179a66d66/06e0c8b4-d6b1-4d54-98b7-ce8ce1af8979)

**Comment by ayushchak** [comment]
What do we do when there is no TTY?

**Comment by msmanak** [comment]
## Slice

[Slice.png](https://images.zenhubusercontent.com/5faabf11b099434179a66d66/c0c4afc7-fdd9-40ff-9ed4-24c8eae1a1cd)


## screen location:


Appointments
- VA Appointment (Past)
- VA Appointment (Cancelled)
- VA Appointment to cancel

- VA Appointment / Cancel/ VA device
- VA Appointment / Cancel/ want to cancel
- VA Appointment / Cancel/ success
- VA Appointment / Cancel/ Call us
- VA Appointment / Cancel/ Something went wrong
- VA Appointment / Cancel/ Automated 

- VAVC-VA location-Appointment (Past)
- VAVC-VA location-Appointment (Canceled)

- VA CC Appointment
- VA CC Appointment (Past)
- VA CC Appointment (Canceled)


Profile
- Letters (Empty)

**Comment by msmanak** [comment]
waiting on @DanielleThierryUSDSVA 's further review and I will update design to reflect this ticket

**Comment by msmanak** [comment]
![Screen Shot 2021-03-10 at 2.13.37 PM.png](https://images.zenhubusercontent.com/5faabf11b099434179a66d66/65112047-8d6c-4ba8-89ef-4348f5a91119)

https://adhoc.invisionapp.com/share/AX108RJZPB6E#/screens/445194522

**Comment by msmanak** [comment]
Final content: "To make a voice or TTY call, tap the first link. If you'd like help to make a TTY call, tap the second link."
